src/routes/index.ts

```diff
@@ -37,9 +39,17 @@ const getContacts = async () => {
 
 router.get('/contacts', async (req, res) => {
 
-    const contact = await getContacts();
-    res.json(contact);
-    console.log('got contacts');
-    res.end();
+    try
+    {
+        const contact = await ApiCall(getContacts,"string");//await getContacts();
+        res.json(contact);
```

Comment:
Error checking if ApiCall returns an error?

Reply:
Could you please give an example of which errors you are referring to please?

Reply:
nvm. It will probably be up to the backend on how they use the function. Looks like at line 215 is wehre you return a 400 error right?

Reply:
Yea line 215 and line 195. the HubApiCall function would return a rejection if any errors occurred and the back end team would decide what to do in their catch, as an example in this instance they were redirected to "/hubspot".